docs/SUMMARY.md

```diff
@@ -41,62 +41,8 @@
-  - [concrete.ml.onnx.onnx_impl_utils.md](references/api/concrete.ml.onnx.onnx_impl_utils.md)
-  - [concrete.ml.onnx.onnx_model_manipulations.md](references/api/concrete.ml.onnx.onnx_model_manipulations.md)
-  - [concrete.ml.onnx.onnx_utils.md](references/api/concrete.ml.onnx.onnx_utils.md)
-  - [concrete.ml.onnx.ops_impl.md](references/api/concrete.ml.onnx.ops_impl.md)
-  - [concrete.ml.pytest.md](references/api/concrete.ml.pytest.md)
-  - [concrete.ml.pytest.torch_models.md](references/api/concrete.ml.pytest.torch_models.md)
-  - [concrete.ml.pytest.utils.md](references/api/concrete.ml.pytest.utils.md)
-  - [concrete.ml.quantization.base_quantized_op.md](references/api/concrete.ml.quantization.base_quantized_op.md)
-  - [concrete.ml.quantization.md](references/api/concrete.ml.quantization.md)
-  - [concrete.ml.quantization.post_training.md](references/api/concrete.ml.quantization.post_training.md)
-  - [concrete.ml.quantization.quantized_module.md](references/api/concrete.ml.quantization.quantized_module.md)
-  - [concrete.ml.quantization.quantized_module_passes.md](references/api/concrete.ml.quantization.quantized_module_passes.md)
-  - [concrete.ml.quantization.quantized_ops.md](references/api/concrete.ml.quantization.quantized_ops.md)
-  - [concrete.ml.quantization.quantizers.md](references/api/concrete.ml.quantization.quantizers.md)
-  - [concrete.ml.search_parameters.md](references/api/concrete.ml.search_parameters.md)
-  - [concrete.ml.search_parameters.p_error_search.md](references/api/concrete.ml.search_parameters.p_error_search.md)
-  - [concrete.ml.sklearn.base.md](references/api/concrete.ml.sklearn.base.md)
-  - [concrete.ml.sklearn.glm.md](references/api/concrete.ml.sklearn.glm.md)
-  - [concrete.ml.sklearn.linear_model.md](references/api/concrete.ml.sklearn.linear_model.md)
-  - [concrete.ml.sklearn.md](references/api/concrete.ml.sklearn.md)
-  - [concrete.ml.sklearn.neighbors.md](references/api/concrete.ml.sklearn.neighbors.md)
-  - [concrete.ml.sklearn.qnn.md](references/api/concrete.ml.sklearn.qnn.md)
-  - [concrete.ml.sklearn.qnn_module.md](references/api/concrete.ml.sklearn.qnn_module.md)
-  - [concrete.ml.sklearn.rf.md](references/api/concrete.ml.sklearn.rf.md)
-  - [concrete.ml.sklearn.svm.md](references/api/concrete.ml.sklearn.svm.md)
-  - [concrete.ml.sklearn.tree.md](references/api/concrete.ml.sklearn.tree.md)
-  - [concrete.ml.sklearn.tree_to_numpy.md](references/api/concrete.ml.sklearn.tree_to_numpy.md)
-  - [concrete.ml.sklearn.xgb.md](references/api/concrete.ml.sklearn.xgb.md)
-  - [concrete.ml.torch.compile.md](references/api/concrete.ml.torch.compile.md)
-  - [concrete.ml.torch.hybrid_model.md](references/api/concrete.ml.torch.hybrid_model.md)
-  - [concrete.ml.torch.md](references/api/concrete.ml.torch.md)
-  - [concrete.ml.torch.numpy_module.md](references/api/concrete.ml.torch.numpy_module.md)
-  - [concrete.ml.version.md](references/api/concrete.ml.version.md)
```

Comment:
not good @andrei-stoian-zama @RomanBredehoft

Reply:
answered by andrei here : https://github.com/zama-ai/concrete-ml/pull/576#discussion_r1549212458

Reply:
hum, is it compatible with what we have for Concrete gitbook and is it what was agreed with Yuxi? 
by the way, if we remove that, pretty sure we can remove corresponding .sh

Reply:
yes it is , basically the API was refered twice in the summary, now it's available in : 
```
## References

- [API](references/api/README.md)
```

like Yuxi did

I'm not sure what you mean by "the corresponding .sh" but isn't it what I did in https://github.com/zama-ai/concrete-ml/pull/580 ?